db/migrate/20200114142128_create_users.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      t.string :login
```

Comment:
You need to add constraint here, not only Rails app must check presence of login, DB must check it too. I suggect to add "null: false" for login and password columns